client/src/auth/requireAuth.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import axios from 'axios';
+
+axios.defaults.baseURL = 'http://localhost:9090/api'
+
+axios.interceptors.request.use(
```

Comment:
This is an interesting way to do it! Another option is using an [axios instance](https://github.com/axios/axios#creating-an-instance) for any base url or authorization stuff. You can also do this same sort of customization on axios instances as well.